src/Controls/src/Core/Handlers/Items/CarouselViewHandler.Windows.cs

```diff
@@ -433,6 +452,14 @@ WSnapPointsAlignment GetWindowsSnapPointsAlignment(SnapPointsAlignment snapPoint
 			return WSnapPointsAlignment.Center;
 		}
 
+		void LayoutPropertyChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
PropertyChanged is never triggered

Reply:
When scrolling the carouselview , the SnapPoint does not respond, it scrolls more than one item